configs/samples/simpleusb.conf.sample

```diff
@@ -106,6 +106,11 @@
 ;ppX=out0					; printer port pin(x) [x is 2-9] initial output state [out0,out1] or 'ptt' (for output pins)
 ;ppX=in						; printer port pin(x) [x is 10,12,13,15] input pin [in,cor,ctcss]
 
+legacyaudioscaling = no     ; If yes, continue to do raw audio sample scaling and clipping, resulting in Tx audio levels increasing
```

Comment:
Add here that the default is currently yes but will eventually be no and/or setting removed

Reply:
Updated this with details on plan to remove the cfg and scaling/clipping.